packages/ckeditor5-table/src/plaintableoutput.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module table/plaintableoutput
+ */
+
+import { Plugin } from 'ckeditor5/src/core';
+import Table from './table';
+import TableCaption from './tablecaption';
+import TablePropertiesEditing from './tableproperties/tablepropertiesediting';
+
+/**
+ * The plain table output feature.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class PlainTableOutput extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'PlainTableOutput';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ Table, TablePropertiesEditing, TableCaption ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+
+		// Override default table data downcast converter.
+		editor.conversion.for( 'dataDowncast' ).elementToStructure( {
+			model: {
+				name: 'table',
+				attributes: [ 'headingRows' ]
```

Comment:
I am wondering why there's `headingRows` attribute here. Could you explain?

Reply:
Well, it was added in order to trigger conversion after `headingRows` attribute change, but it actually turned out it's unnecessary. It's being handled by another converter in the main `Table` plugin.